pkg/networkplugin-syncer/handlers/ovn/handler.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/submariner-io/submariner/pkg/networkplugin-syncer/handlers/ovn/nbctl"
 )
 
+//nolint:stylecheck // Limit line length as per lll
```

Comment:
The "_ST1005: error strings should not end with punctuation or a newline_" error is actually due to the ending period and not line-length. So we just need to remove the period 😄